hcache/compr_zstd.c

```diff
@@ -120,6 +120,9 @@ static void *compr_zstd_decompress(void *cctx, const char *cbuf, size_t clen)
 {
   struct ComprZstdCtx *ctx = cctx;
 
+  if (!cctx || clen < 8)
```

Comment:
Why do you check for `clen < 8`? How can you even get there?

Reply:
Overseen ... also the lz4 <0 thing.